Applications/Utils/MeshEdit/CMakeLists.txt

```diff
@@ -112,6 +112,17 @@ target_link_libraries (reviseMesh
 	${CATALYST_LIBRARIES}
 )
 ADD_CATALYST_DEPENDENCY(reviseMesh)
+set_target_properties(reviseMesh
+	PROPERTIES FOLDER Utilities)
 
-set_target_properties(reviseMesh PROPERTIES FOLDER Utilities)
+add_executable( ResetPropertiesInPolygonalRegion
+	ResetPropertiesInPolygonalRegion.cpp )
+target_link_libraries( ResetPropertiesInPolygonalRegion
```

Comment:
Recently I removed extraneous whitespace in CMake-files for consistency. Please remove the whitespace before the first parameter.